backend/dal/Migrations/v01.13.01/Up/PostUp/01-NotificationTemplates.sql

```diff
@@ -0,0 +1,6 @@
+PRINT 'Updating NotificationTemplates'
+
+-- Include the CITZ_RPD_IMIT_HELP@gov.bc.ca email to the Access Request notification
+UPDATE dbo.[NotificationTemplates]
+SET [To] = 'CITZ_RPD_IMIT_HELP@gov.bc.ca'
```

Comment:
Make sure this doesn't result in sending multiple copies of this email when it attempts to perform this action.  I think it will because the code looks for all system admins and sends them an email.

Reply:
It looks like I'll need to remove the migration as setting the email in that template will cause the RPD email to get multiple copies of the access request.

I can make a change to the api in the UserController.cs file and I guess I'll need to hardcode the email address into the AddAccessRequestAsync function.

Reply:
You shouldn't need to hardcode it.  Once you have updated the template (as you have in this PR) all you should need to do is change the code to only send one email instead of one for each system admin.